data/publiccloud/azure_runcmd.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash -u
+set -o pipefail
+. $(dirname "${BASH_SOURCE[0]}")/azure_lib_fn.sh
+# Create a vm and use run-command to execute RunShellscript
+# list, show , delete the created resource
+# set -x
```

Comment:
was this on purpose ? I mean I don't mind but again commented code should not be merged . so or uncomment this line or delete it